content/300-accelerate/500-known-limitations.mdx

```diff
@@ -19,3 +19,7 @@ At the moment, it is not possible to cache the responses of [raw queries](/orm/p
 ## Not compatible with the fluent API
 
 Client Extensions (which are used in Accelerate) currently do not correctly forward the [fluent API](/orm/prisma-client/queries/relation-queries#fluent-api) types. We hope to get a fix into Client Extensions soon.
+
+## Limits on programmatic cache invalidation
```

Comment:
I think this is rather a limit, not a limitation (= shortcoming). I'd add this information to the `$accelerate.invalidate` API reference section